package.json

```diff
@@ -48,5 +48,9 @@
   "collective": {
     "type": "opencollective",
     "url": "https://opencollective.com/parcel"
+  },
+  "dependencies": {
+    "@babel/core": "^7.5.5",
```

Comment:
I think if we use the `@parcel/babel-register` package we shouldn't need these here.